backend/test/til_web/controllers/activities/reaction_controller_test.exs

```diff
@@ -0,0 +1,174 @@
+defmodule TilWeb.Activities.ReactionControllerTest do
+  use TilWeb.ConnCase
+  import Til.Guardian
+  import Til.Factory
+  alias Til.ShareableContent
+  alias Til.Repo
+  alias Til.Activities.Reaction
+
+  describe "POST /api/posts/reactions" do
+    test "creates reaction given type for post when not reacted yet and returns ok", %{conn: conn} do
+      current_user = insert(:user)
+      {:ok, token, _} = encode_and_sign(current_user.uuid, %{})
+      type = "like"
+
+      post = insert(:post)
+      insert(:reaction, user_id: current_user.id, post_id: post.id, type: "love")
+
+      response =
+        conn
+        |> put_req_header("authorization", "bearer: " <> token)
+        |> post(Routes.post_reaction_path(conn, :react, post.id), %{
+          type: type
+        })
+
+      assert response.status == 200
+
+      reactions = Repo.all(Reaction)
+
+      assert length(reactions) == 2
+
+      [_, reaction] = reactions
+
+      assert reaction.user_id == current_user.id
+      assert reaction.post_id == post.id
+      assert reaction.type == type
```

Comment:
it's better to be explicit in tests for the sake of readability (no need for DRY code), so you can remove 'type' variable and put `"like"` everywhere